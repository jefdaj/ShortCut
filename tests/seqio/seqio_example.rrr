result = "seqio example not written yet"
