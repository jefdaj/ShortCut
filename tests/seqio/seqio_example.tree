.
|-- cache
|-- exprs
|   `-- str
|       `-- 8e9a74b8b4.str -> ../../cache/lines/...
`-- vars
    `-- result -> ../exprs/str/8e9a74b8b4.str

4 directories, 2 files
