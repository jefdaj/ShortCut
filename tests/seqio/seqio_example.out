"seqio example not written yet"
